frame/system/src/lib.rs

```diff
@@ -485,19 +485,35 @@ decl_module! {
 		}
 
 		/// Make some on-chain remark.
+		/// 
+		/// # <weight>
+		/// - `O(1)`
+		/// # </weight>
 		#[weight = SimpleDispatchInfo::FixedNormal(MINIMUM_WEIGHT)]
 		fn remark(origin, _remark: Vec<u8>) {
 			ensure_signed(origin)?;
 		}
 
 		/// Set the number of pages in the WebAssembly environment's heap.
+		/// 
+		/// # <weight>
+		/// - `O(1)`
+		/// - 1 storage write.
+		/// # </weight>
 		#[weight = SimpleDispatchInfo::FixedOperational(MINIMUM_WEIGHT)]
 		fn set_heap_pages(origin, pages: u64) {
 			ensure_root(origin)?;
 			storage::unhashed::put_raw(well_known_keys::HEAP_PAGES, &pages.encode());
 		}
 
 		/// Set the new runtime code.
+		/// 
+		/// # <weight>
+		/// - `O(C + S)` where `C` length of `code` and `S` complexity of `can_set_code`
```

Comment:
can_set_code is part of frame_system, shouldn't its weight be precised here ?

I think only external/generic cost should be abstracted no ?